vpr/src/pack/cluster.cpp

```diff
@@ -1233,6 +1243,24 @@ static enum e_block_pack_status try_pack_molecule(t_cluster_placement_stats* clu
         return BLK_FAILED_FEASIBLE;
     }
 
+    //for loop to check if cluster PartitionRegion intersects with atom PartitionRegion
```

Comment:
check if every atom in the molecule is legal in the cluster from a floorplanning perspective